bootstrap/internal/controllers/rke2config_controller.go

```diff
@@ -455,6 +547,10 @@ func (r *RKE2ConfigReconciler) joinWorker(ctx context.Context, scope *Scope) (re
 		Permissions: filePermissions,
 	}
 
+	files, err := r.generateFileListIncludingRegistries(scope, ctx, configFiles)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
 	// TODO: Implement adding additional files through API
```

Comment:
Same comment as previously.

Reply:
Same, it is implemented in the function above.